math/rate.go

```diff
@@ -1,59 +0,0 @@
-package math
-
-import (
-	"sync"
-	"time"
-
-	"go.uber.org/atomic"
-)
-
-// EwmaRate tracks an exponentially weighted moving average of a per-second rate.
-type EwmaRate struct {
```

Comment:
Hmm, this actually looks useful across different projects.

Reply:
(I think we can remove it from now, and put it back with improved package structure in the future, if we want it).